client/src/reader/components/annotation/popup/menus/Main/Items/CurrentReadingGroup.js

```diff
@@ -37,9 +41,14 @@ function CurrentReadingGroup({
         data-name="readingGroup"
         className="annotation-popup__button annotation-popup__button--stacked annotation-popup__button--secondary-dark"
       >
-        <span className="screen-reader-text">Select reading group</span>
+        <span className="screen-reader-text">
+          {t("actions.select_item", { item: t("glossary.reading_group_one") })}
```

Comment:
Interpolation of keys should only happen inside the translation file. So, you might need to add a "select_reading_group" key in addition to "select_item":
```
"select_reading_group": "Select $t(glossary.reading_group_one)"
```